src/api.js

```diff
@@ -277,8 +277,10 @@ function qmlweb_parse($TEXT, document_type, exigent_mode) {
       }
     } else if (S.token.type == "name") {
       if (S.token.value == "property" && (S.token.type == "name" || S.token.value == "var")) {
-        next();
-        return qmlpropdef();
+        if (!is_token(peek(), "punc", ":")) {
```

Comment:
Hm. Linter didn't catch this for some reason, but there is no need for nested ifs here, just use a single one:
```js
      if (S.token.value == "property" &&
          (S.token.type == "name" || S.token.value == "var") &&
          !is_token(peek(), "punc", ":")) {
        next();
        return qmlpropdef();
      }
```

Upd: ah, we don't have linter enabled in this repo.